site/tests/integration/blockpage.test.ts

```diff
@@ -84,7 +84,9 @@ test("Block page should contain key elements", async ({
   if (browserName !== "webkit") {
     await expect(
       page.frameLocator("iframe[title='block']").locator("input"),
-    ).toBeVisible();
+    ).toBeVisible({
+      timeout: 30000, // @todo Remove after re-engineering block sandbox
```

Comment:
I matched the timeout with the one we have in universal tests after noticing that 5s is not always enough for esm.sh CDN.